.changeset/loud-ghosts-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': fix
```

Comment:
fyi, this needed to be `patch`. releases are failing because this is an invalid value: https://github.com/sveltejs/kit/pull/9365